src/officehours_ui/templates/officehours_ui/auth_prompt.html

```diff
@@ -5,6 +5,7 @@
 <div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
   <h1>Allow Access</h1>
   <p>Please click the following link to allow Remote Office Hours Queue to access {{ backend_friendly_name }}.</p>
```

Comment:
Change "Please click the following link" to "Please click Authorize"

Reply:
Good point, done.